js/client-chat.js

```diff
@@ -1237,6 +1237,20 @@
 					this.$chat.append('<div class="notice">' + Tools.sanitizeHTML(row.slice(1).join('|')) + '</div>');
 					break;
 
+				case 'globalnotify':
+					app.rooms[''].addPM('~', '/raw ' + row.slice(1).join('|'), '' + this.users[app.user.changed.userid]);
```

Comment:
This one's inconsistent now, anyway.

Just make global declares show up as PMs server-side.